EnglishFilter/sections/specific.txt

```diff
@@ -291,6 +291,7 @@ techpowerup.com##.site-headr__logo-wrapper + * a[href*="/images/"] img
 techpowerup.com##div[class*="bar-it"] a[href*="/images/"] img
 techpowerup.com#%#//scriptlet('remove-attr', 'wdrwgbethte', 'div')
 !
+viva.pl##.pending-ad
```

Comment:
add also `viva.pl##.placement` to english > foreign > polish

and `viva.pl##.share-photo` to social > specific

Reply:
Fixed.